libs/shared/src/types/overlay/response.ts

```diff
@@ -30,5 +35,9 @@ export interface CreatePlaybackConversationResponse {
 }
 
 export interface ExportConversationResponse {
+  exportConversation: LatestExportConversationsFormat;
```

Comment:
Maybe it make sense to use old property name `conversation` for consistency with other response interfaces
```suggestion
  conversation: LatestExportConversationsFormat;
```

Reply:
This is exactly why I changed this - I don't want to make an expression that returned object is same object format as in other APIs - it return special export format object to download on client side